projects/ui/src/lib/components/po-page/po-page-list/po-page-list.component.ts

```diff
@@ -150,7 +152,7 @@ export class PoPageListComponent extends PoPageListBaseComponent
   }
 
   callActionFilter(field: string): void {
-    this.callFunction(this.filter[field], this.parentRef);
+    this.callFunction(this.filter[field], this.parentRef, this.filterInput.nativeElement.value);
```

Comment:
Como o parentRef no caso de funções sera removido, da pra remover a parte de callFunction do page-list

neste caso ficaria assim:`
this.filter[field](this.filterInput.nativeElement.value);`

Reply:
Linha 101 também